lua/luasnip/init.lua

```diff
@@ -281,6 +281,37 @@ local function snip_expand(snippet, opts)
 	return snip
 end
 
+---Try to expand a snippet at the current cursor position.
+local function snip_try_expand(snippet, opts)
```

Comment:
Cool idea!
Although I'd frame it not like "try to match the snippet here and expand if it works", but as "okay since the snippet may want to retrieve some additional data and put that into `expand_params`, here's this helper which retrieves all that".
Basically, I'm proposing to change the name to "snip_match_expand" xD